### DISCUSSION TITLE
Fock latest stable vscpd release from git

### DISCUSSION THREAD

**Discussion by dinguluer** [grodansparadis/vscp#273] [Q&A]
Hi,

How to  Fock latest stable vscpd release from git ?
Any stable branch for  following release ?

![image](https://github.com/grodansparadis/vscp/assets/370819/74125472-cc0a-4b2a-b62d-9c7d182bd747)

If i am correct then this will be unstable :--
https://github.com/grodansparadis/vscp

Regards,
DInesh

**Comment by grodansparadis** [comment]
There are two branches. "Master" and "development".  The development branch is cutting edge. The maste branch is more or less "stable" but may have issues. So fork master branch if you want code that give you least problems.
